src/Task/index.js

```diff
@@ -34,23 +35,22 @@ const TaskSumary = ({ title, htmlTitle, variant }) => {
   )
 }
 
-const Task = ({ title, htmlTitle, variant, children }) => {
-
+const Task = ({ title, htmlTitle, variant, children, extra }) => {
   const hasChildren = React.Children.count(children) > 0
   return hasChildren ?
     (
-      <details className={cx('ola_task', variant && `is-${variant}`)}>
-        <TaskSumary title={title} htmlTitle={htmlTitle} variant={variant} />
+      <details className={cx('ola_task', variant && `is-${variant}`, extra && 'is-extra')}>
```

Comment:
I think we don't need the `is-extra` class here. If the task has extra, it's only about adding one more element to to `ola_task-header`.

Reply:
done